README.md

```diff
@@ -631,7 +631,7 @@ This API allows you to check whether the given element has a text content or not
 ##### Using document.querySelector
 
 ```javascript
-const button = document.querySelector('[data-testid="count-value"]')
+const content = document.querySelector('[data-testid="count-value"]')
```

Comment:
Thanks for spotting this inconsistency. But now that we're at it, the name `content` is not good either. Can you change it to `element` and also use that in the `expect` assertions below?

Reply:
Good call — updated in 0b887ac.